### ISSUE TITLE
Implementation suggestion of additional solver

### ISSUE THREAD

**Issue by Emmanuel-R8** [SciML/DiffEqFlux.jl#440]
Hypersolvers: Toward Fast Continuous-Depth Models

[https://arxiv.org/abs/2007.09601]()

**Comment by ChrisRackauckas** [comment]
Yeah, it's the same method as the one people have been discussing at conferences:

https://arxiv.org/abs/2003.09573

I plan to get some high performance implementations of these sometime next summer.
